### PR TITLE
New resource link

### PR DESCRIPTION
**Pull request by swissjake**

### CODE REVIEW

File: database/frontend/images.json [hunk 1]
```diff
@@ -25,5 +25,20 @@
...
+    {
+        "name": "Picjumbo",
+        "description": "Download free stock photos, backgrounds and free high-resolution images for personal and commercial use",
+        "url": "https://picjumbo.com/",
+        "category": "frontend",
+        "subcategory": "images"
+    },
+    {
+        "name": "Shotstash",
+        "description": "Free HD Stock Pictures for Creative Professionals",
```
Comment (rupali-codes) [inline database/frontend/images.json:38]: hi, could you please add a little more description for it?

File: database/frontend/images.json [hunk 2]
```diff
@@ -25,5 +25,20 @@
         "url": "https://kaboompics.com/",
         "category": "frontend",
         "subcategory": "images"
+    },
+    {
+        "name": "Picjumbo",
+        "description": "Download free stock photos, backgrounds and free high-resolution images for personal and commercial use",
```
Comment (CBID2) [inline database/frontend/images.json:31]: Maybe you can discuss the benefits of this source?

File: database/frontend/images.json (hunk 1 above)
Comment (CBID2) [reply database/frontend/images.json:38]: I agree with @rupali-codes @swissjake. Maybe you can describe its advantages?

File: database/frontend/images.json (hunk 2 above)
Comment (rupali-codes) [reply database/frontend/images.json:31]: Yeah, should be at least 15 words long. thanks chrissy :)

**Review by rupali-codes — APPROVED** [review]
thanks

File: database/frontend/images.json [hunk 3]
```diff
@@ -25,5 +25,20 @@
         "url": "https://kaboompics.com/",
         "category": "frontend",
         "subcategory": "images"
```
Comment (CBID2) [inline database/frontend/images.json:27]: This could use a description (e.g., the benefits of using this tool). What do you think @rupali-codes?

Comment (rupali-codes) [reply database/frontend/images.json:27]: yeah, but since it's for images, what could be the benefits? 😅

**Pull request merged**